modules/monitoring_platform/eks.tf

```
@@ -33,4 +33,18 @@ resource "aws_iam_role_policy_attachment" "s3_access_policy_attachment" {
   policy_arn = aws_iam_policy.s3_access_policy.arn
   role       = module.monitoring_alerting_cluster.worker_iam_role_name
   count      = var.is_eks_enabled ? 1 : 0
+}
+
+resource "aws_iam_policy" "kms_access_policy" {
+  name = "${var.prefix_pttp}-thanos-prometheus-kms-policy"
```

```suggestion
  name = "${var.prefix}-thanos-prometheus-kms-policy"
```

prefix_pttp is outdated and we should be moving away from it where possible